### PR TITLE
[digitalstrom] Update CODEOWNERS

### PR DESCRIPTION
**Pull request by lolodomo**
As discussed in #12033 , the mentioned authors for the digitalstrom binding will no more contribute or review.

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by jlaur** [comment]
@lolodomo - do we need valid sign-off for this?

**Review by jlaur — APPROVED** [review]

**Comment by lolodomo** [comment]
I do not understand what's wrong with my signed-off-by. 
Maybe because I edited the file directly in github? 
You can merge IMHO.

**Comment by jlaur** [comment]
> Maybe because I edited the file directly in github?

Might be. It detected difference in real name, probably because username was used as real name in the commit? I'll merge.

Commit sha: 42e8a81, Author: lolodomo, Committer: GitHub; Expected "**lolodomo** lg.hc@free.fr", but got "**Laurent Garnier** lg.hc@free.fr".

**Comment by jlaur** [comment]
@lolodomo - to me it would make sense not to add milestone to such PR as it's not a code change, but not sure how strict this rule is to be followed?

**Comment by lolodomo** [comment]
I agree with you.

**Comment by lolodomo** [comment]
@wborn : I see more and more PRs with tags "bug" and "enhancement". Is it a problem to produce the release notes? Should we avoid that or is it ok?

**Comment by kaikreuzer** [comment]
@lolodomo No, release notes only look at PRs, issues are ignored, so it doesn't matter what tags they have.

**Comment by lolodomo** [comment]
I was not clear, I am talking about PR having labels "bug"  and "enhancement" on the same PR.

**Comment by wborn** [comment]
If you assign both "bug" and "enhancement" the PR will probably just be added under "Enhancements" in the release notes.

That's because [the script assigns](https://github.com/BClark09/openhab-scripts/blob/24593675bf26d291860b582dc20ecae6ef71ec16/build.gradle#L162-L170) one `pullType` per PR and the GitHub API used for getting the PR info seems to return them sorted.

See:

https://api.github.com/search/issues?q=repo:openhab/openhab-addons%20is:pr%20milestone:%223.3%22&per_page=100

```
"labels": [
    {
        "id": 99327134,
        "node_id": "MDU6TGFiZWw5OTMyNzEzNA==",
        "url": "https://api.github.com/repos/openhab/openhab-addons/labels/bug",
        "name": "bug",
        "color": "fc2929",
        "default": true,
        "description": "An unexpected problem or unintended behavior of an add-on"
    },
    {
        "id": 99327136,
        "node_id": "MDU6TGFiZWw5OTMyNzEzNg==",
        "url": "https://api.github.com/repos/openhab/openhab-addons/labels/enhancement",
        "name": "enhancement",
        "color": "84b6eb",
        "default": true,
        "description": "An enhancement or new feature for an existing add-on"
    }
],
```

IMHO an enhancement can also fix bugs besides implementing new features, whereas bugfixes only fix bugs. So I'd only use the "enhancement" label for those.

**Comment by lolodomo** [comment]
Thank you @wborn for the explanation. 
I am going to update merged PRs in 3.3 having both labels

**Pull request merged**
